spark/src/main/scala/org/apache/spark/sql/delta/TypeWideningMetadata.scala

```diff
@@ -54,6 +56,8 @@ private[delta] case class TypeChange(
 }
 
 private[delta] object TypeChange {
+  // tableVersion was a field present during the preview and removed afterwards. We preserve it if
+  // it's already present in the type change metadata of the table to avoid breaking older clients.
```

Comment:
What do they still use it for? Would it be possible to always set it to some constant number (e.g. zero)?